### ISSUE TITLE
Should not be using remembered Eve location on BS2.

### ISSUE THREAD

**Issue by Tarr-OHOL** [jasonrohrer/OneLife#502]
I believe this is the culprit of what occurred after the server reset just the other day. Someone ended up spawning in the bell town due to the eve respawn mechanic used for low population play (when the server was just coming back up the population was low enough) which proceeded to dump one lineage far away from everyone else. 

This mechanic completely goes against the last updates work to bring players together and as such should be disabled for BS2 exclusively. 

TL;DR: Disable Eve remembered spawns for BS2 as it displaced one lineage 2k+ away from the other three.
